File: src/kyu_7/minimize_sum_of_array_array_series_number_1/cpp/minimize_sum_of_array_array_series_number_1.cpp
#include <vector>
using namespace std;

int minSum (vector<int>passed)
{
  std::sort(passed.begin(), passed.end());
  int result = 0;
  for (int i = 0 ; i < passed.size() / 2; i++)
  {
    result += passed[i] * passed[passed.size() - i - 1];
  }
  return result;
}

File: src/kyu_7/minimize_sum_of_array_array_series_number_1/csharp/minimize_sum_of_array_array_series_number_1.cs
using System;

class Kata
{
    public static int MinSum(int[] a)
    {
        int result = 0;
        Array.Sort(a);
        for (int i = 0, j = a.Length - 1; i < a.Length; i += 2, j -= 2) {
            result += a[j] * a[i];
        }
        return result;
    }
}